sharness/lib/test-lib.sh

```diff
@@ -121,7 +121,7 @@ cluster_kill(){
 }
 
 cluster_start(){
-    ipfs-cluster-service --config "test-config" daemon >"$IPFS_OUTPUT" 2>&1 &
+    ipfs-cluster-service --config "test-config" daemon --consensus crdt >"$IPFS_OUTPUT" 2>&1 &
```

Comment:
Does this mean that we are no longer performing the sharness tests with `raft`? If we are maintaining that both `crdt` and `raft` are viable consensus components, then we should continue to test both as fully as we can?

Reply:
sharness tests run on single-peer so it does not matter much. I agree the export/import tests should be done with those, as that's actually something sharning IS testing and has two distinct codepaths.